### ISSUE TITLE
command to list available local images

### ISSUE THREAD

**Issue by hatched** [CanonicalLtd/multipass#140]
It would be nice if there was a command to list the local images `multipass images` instead of having to run `multipass info <vm>` and copy the hash.

Adding that those images get purged after N days, the selected hash may not be available anyways and it'll start the image download process.

**Comment by albaguirre** [comment]
@hatched, Is this to workaround multipassd downloading a newer image?

Would a flag to multipass launch be better? maybe like multipass launch --cached?
@Saviq ^

**Comment by hatched** [comment]
Yes, I wanted to prevent a new image from being downloaded if there already was one on the system as it was a poor connection.

**Comment by Saviq** [comment]
Yeah I would be OK with adding a `--cached` flag to `launch` and `find` commands, will have to confer with UX design for this, though.

**Comment by panlinux** [comment]
I just experienced the same need. I have a bionic vm running, that I launched earlier today, and now when I launched another bionic vm, multipass started downloading it again.

**Comment by panktrip** [comment]
This feature would be really helpful.  we should be able to list all the images that we already downloaded using Multipass.  pretty similar to how vagrant shows the list of box/images  (using vagrant box list  command).

**Comment by jhcook** [comment]
Please add this feature!

**Comment by billhong-just** [comment]
Please add this feature!

**Comment by sharder996** [comment]
Before implementing this we would like to remove the necessity to be connected to the internet for Multipass to operate. Multipass currently operates on this assumption and is something that needs to be addressed before we add this behavior.
